sys/include/cb_mux.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2018 Acutam Automation, LLC
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   sys_cb_mux
```

Comment:
this doxygen group is not defined. you can use a `@defgroup` above this line and maybe also add a bit of documentation.

Reply:
```@defgroup``` and summary added.